### PR TITLE
Styled components for contract list, contract item, and contract form…

### PR DESCRIPTION
**Pull request by brianjcarroll**
… pages

### CODE REVIEW

**Review by ChristianChiarulli — APPROVED** [review]
LGTM

File: src/pages/ContractPage.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      <div>by {author}</div>
+      <Card className="">
+        <CardHeader>
+          <CardTitle>{title}</CardTitle>
+        </CardHeader>
+        <CardContent>
+          <div>{description}</div>
+          {/* <div>Expires on: {expiration}</div>
+            <div>Created on: {createdOn}</div>
+            <div>ID: {id}</div> */}
```
Comment (tvolk131) [inline src/pages/ContractPage.tsx:118]: Please remove

File: src/pages/ContractsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from "~/components/ui/card";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "~/components/ui/tabs";
+
+// import { invoke } from "@tauri-apps/api/tauri";
+
+// import { invoke } from "@tauri-apps/api/primitives";
```
Comment (tvolk131) [inline src/pages/ContractsPage.tsx:14]: Please remove

File: src/pages/ContractsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+} from "react-router-dom";
+
+// import { invoke } from "@tauri-apps/api/tauri";
+
+async function setSecret() {
+  // Learn more about Tauri commands at https://tauri.app/v1/guides/features/command
+  // const test = await invoke("set_secret");
+  // console.log(test);
+  // alert("hello");
+}
```
Comment (tvolk131) [inline src/pages/ContractsPage.tsx:32]: Please remove

File: src/pages/ContractsPage.tsx [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+export default function ContractsPage() {
+  return (
+    <>
+      <div className="flex-col md:flex">
+        <div className="flex-1 space-y-4 p-8 pt-6">
+          <div className="flex items-center justify-between space-y-2">
+            <h2 className="text-3xl font-bold tracking-tight">Contracts</h2>
+            <div className="flex items-center space-x-2">
+              {/* <CalendarDateRangePicker /> */}
```
Comment (tvolk131) [inline src/pages/ContractsPage.tsx:42]: Please remove

File: src/pages/ContractsPage.tsx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+              <Button
+
+              // onClick={setSecret}
```
Comment (tvolk131) [inline src/pages/ContractsPage.tsx:45]: Please remove

File: src/pages/ContractsPage.tsx [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+              </Button>
+            </div>
+          </div>
+          <Tabs defaultValue="overview" className="space-y-4">
+            <TabsList>
+              <TabsTrigger value="overview">Overview</TabsTrigger>
+              <TabsTrigger value="sent">Sent</TabsTrigger>
+              <TabsTrigger value="received">Received</TabsTrigger>
+              {/* <TabsTrigger value="accepted">Accepted</TabsTrigger> */}
+              {/* <TabsTrigger value="rejected">Rejected</TabsTrigger> */}
```
Comment (tvolk131) [inline src/pages/ContractsPage.tsx:57]: Please remove

File: src/pages/ContractsPage.tsx [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+              <TabsTrigger value="archive">Complete</TabsTrigger>
+            </TabsList>
+            <TabsContent value="overview" className="space-y-4">
+              <div className="flex flex-col">
+                <Contracts />
+              </div>
+            </TabsContent>
+            <TabsContent value="sent" className="space-y-4">
+              <div className="flex flex-col">
+                {/* this is list of cards*/}
```
Comment (tvolk131) [inline src/pages/ContractsPage.tsx:67]: This comment is low value, please remove

File: src/pages/ContractsPage.tsx [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+                <Card>
+                  <CardHeader>
+                    <CardTitle>Overview</CardTitle>
+                  </CardHeader>
+                  <CardContent className="pl-2">
+                    sent
+                    {/* <Overview /> */}
```
Comment (tvolk131) [inline src/pages/ContractsPage.tsx:74]: Please remove

File: src/pages/CreateContractPage.tsx [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+    resolver: zodResolver(formSchema),
+    defaultValues: {
+      title: "",
+      description: "",
+      cost: 0,
+      expiration: new Date(),
+    },
+  });
+  function onSubmit(values: z.infer<typeof formSchema>) {
+    // Do something with the form values.
```
Comment (tvolk131) [inline src/pages/CreateContractPage.tsx:55]: Can we make this a TODO comment?

`// TODO: Use form values instead of logging them.`

File: src/pages/CreateContractPage.tsx [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+    // ✅ This will be type-safe and validated.
```
Comment (tvolk131) [inline src/pages/CreateContractPage.tsx:56]: What exactly is this comment expressing? Can you rephrase/clarify or remove it?

File: src/pages/LoginPage.tsx [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+  CardTitle,
+} from "~/components/ui/card";
+import { Input } from "~/components/ui/input";
+import { Label } from "~/components/ui/label";
+
+import { generatePrivateKey, getPublicKey, nip19 } from "nostr-tools";
+
+export default function LoginPage() {
+  async function setSecret() {
+    const sk = generatePrivateKey(); // `sk` is a hex string
```
Comment (tvolk131) [inline src/pages/LoginPage.tsx:18]: Can we change the comment here? Something like:

`// Hex string of private key.`
